compiler/codegen/symbol.cpp

```diff
@@ -60,6 +60,28 @@
 *                                                                   *
 ********************************* | ********************************/
 
+char llvmFuncDumpName[FUNC_NAME_MAX+1] = "";
+int llvmFuncOptDump = 0;
+
+void llvmFunctionDump(int optLevel, const std::string &name) {
```

Comment:
I think this function has the wrong arguments. I think it should accept a Chapel Function* and an llvm::Function*.
Then, compare llvmFuncDumpName against chapelFunction->name and chapelFunction->cname.